[tutorial] add tictactoc publish script (#3540)
MystenLabs/sui: doc/src/explore/tutorials.md

remainder of this tutorial.

This tutorial models gas fees under a simplified schema. In practice, the Sui
network charges for gas using its native currency SUI. This transaction fee
equals the computational effort of executing operations on the Sui network (i.e.
gas units) times the price of gas in the SUI currency (i.e. the gas price).

When you complete the setup steps, you can use the either use following script to publish and run the sample code, or perform each step manually. Using the script is optional. To manually run each step, follow the steps starting in the [Gather accounts and gas objects](#gather-accounts-and-gas-objects) section.

## Quick script

> If you prefer not to enter command step by step, or need to go though it multiple
> times for whatever reason (change some move source code ), this automated script
> may be useful to save some time.

Reviewer: ```suggestion
If you prefer not to enter command step by step, or need to go though it multiple
times (such as when you change some Move source code), the following automated script
may be useful to save some time.
```

Reviewer: if nothing else, please fix the extra space in `code )`

equals the computational effort of executing operations on the Sui network (i.e.
gas units) times the price of gas in the SUI currency (i.e. the gas price).

## Quick script
If you prefer not to enter command step by step, or need to go though it multiple
times for whatever reason (change some move source code ), this automated script
may be useful to save some time.
Run this script from the project repo root.
```sh
#!/bin/bash
# a bash script to automate the process of publishing the game package
# this script should be run at root of the repo

# assign address
CLIENT_ADDRESS=$(sui client addresses | tail -n +2)
echo $CLIENT_ADDRESS
ADMIN=$CLIENT_ADDRESS | head -n 1

> PLAYER_X=$CLIENT_ADDRESS | head -n 2

Author: From my view, they seem use PLAYER_X and PLAYER_O? could you double check?

> PLAYER_X=$CLIENT_ADDRESS | head -n 2

Author: updated
